frontend/src/components/SelectApplication.vue

```diff
@@ -26,45 +37,49 @@ onMounted(async () => {
         return Promise.reject(error);
     }
 });
+
+function filterEnv(event: string) {
```

Comment:
Hi Nick, shall we not call it event? even sounds like an event, not a string :  )) thanks!!

Reply:
Thanks. This has been changed.